app/helpers/otp_helper.rb

```diff
@@ -0,0 +1,53 @@
+module OtpHelper
+  def phone_confirmation_instructions
```

Comment:
Given the number of conditionals in this file, I think you might want a set of presenters that confrm to the sae API rather than a single module. Maybe something similar to what Zach did here: https://github.com/18F/identity-idp/pull/853/files